distribution/src/resources/config-tool/default.json

```diff
@@ -430,8 +430,8 @@
   "management_api.handler.token_store_config.remove_oldest_token_on_overflow": "true",
   "management_api.handler.token_config.expiry": "3600",
   "management_api.handler.token_config.size": "2048",
-  "management_api.handler.file_user_store.enable": true,
-  "management_api.handler.user_store.users": [
+  "management_api.file_user_store.enabled": true,
```

Comment:
Let's make this also as **enable** to be consistence with others ?

Reply:
done